elife/config/etc-init.d-goaws

```diff
@@ -0,0 +1,85 @@
+#!/bin/sh
+
+### BEGIN INIT INFO
+# Provides:          goawsdaemon
+# Required-Start:    $local_fs $network $syslog
+# Required-Stop:     $local_fs $network $syslog
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: GoAWS
+# Description:       Start/stop daemon for GoAWS.
+### END INIT INFO
+
+NAME="goaws"
+PIDFILE="/var/run/goaws/goaws.pid"
+PATH="/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin"
+APPDIR="/"
+APPBIN="/usr/local/bin/goaws"
+
+# Include functions
+set -e
+. /lib/lsb/init-functions
+
+start() {
+  printf "Starting '$NAME'... "
+  start-stop-daemon --start --background -m --oknodo --pidfile ${PIDFILE} --make-pidfile --chuid root:root --exec ${APPBIN}
+  aws sns create-topic --name=annual-reports --endpoint=http://localhost:4100
```

Comment:
do we actually need to create all the topics and queues every time this is started or is it something we could do in a `.sls` file with a Salt state?